index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		 <link rel="stylesheet" type ="text/css" href ="css/style.css">
+		<title>Intuit Accounts - Sign Up</title>
+	</head>
+	<body>
+<header>
+
+	<nav>
+		<div class="nav-text">
+			<p><ts style="color:black;">Already have an intuit account?</ts> <br> I forgot my User ID or Password </p>
+		</div>	
+		<div class="nav-button">
+			<span class="ius-white-lock-img"></span><button name="signin" type="signin" class="button" ><img src="images/lock.png" alt="lock no found" /> &nbsp&nbsp<b>Sign in</b></button>
+		</div>
+		
+	</nav>
+</header>	
+<main>
+	<img class="logo" src="images/logo.png" alt="intuit logo">
+</main>
```

Comment:
`REQUIRED` 

Consider using `<main>` to cover the main section of the page, that is extending it until the `<footer>` element, this shows the dominant content of the `<body>` of a document. More on `<main>` [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main).